bin/pt-online-schema-change

```diff
@@ -3468,6 +3468,22 @@ sub check_table {
    }
    my ($dbh, $db, $tbl) = @args{@required_args};
    my $q      = $self->{Quoter} || 'Quoter';
+
+   # https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html
+   # MySQL may use use case-insensitive table lookup, this is controller by
+   # @@lower_case_table_names. 0 means case sensitive search, 1 or 2 means
+   # case insensitive lookup.
+
+   my $lctn_sql = 'SELECT @@lower_case_table_names';
+   PTDEBUG && _d($lctn_sql);
+   my ($lower_case_table_names) = $dbh->selectrow_array($lctn_sql);
+   PTDEBUG && _d("lower_case_table_names=$lower_case_table_names");
+
+   if ($lower_case_table_names > 0) {
+       PTDEBUG && _d("MySQL uses case-insensitive lookup, converting '$tbl' to lowercase");
+       $tbl = lc $tbl;
+   }
+
```

Comment:
This code is in the TableParser package and should be defined in lib/TableParser.pm Otherwise, it will be overwritten once we update the modules. Please adjust your PR.